pkgs/by-name/ya/yara-x/package.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, cmake
+, installShellFiles
+, testers
+, yara-x
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "yara-x";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "VirusTotal";
+    repo = "yara-x";
+    rev = "v${version}";
```

r-ryantm will update it with its first update.

```suggestion
    rev = "refs/tags/v${version}";
```